src/AI/AIUtils.java

```diff
@@ -116,8 +117,8 @@ public static Map<UnitActionFactory, ArrayList<GameAction> > getAvailableUnitAct
   }
 
   /**
-   * Find locations of every property that is not owned by myCo or an ally.
-   * @param myCo Properties of this Commander and allies will be ignored.
+   * Find locations of every property that is not owned by myArmy or an ally.
+   * @param myArmy Properties of this Commander and allies will be ignored.
```

Comment:
Comment says it takes an army now, but it still takes a Commander. Given that everywhere it's invoked it's given `army.cos[0]`, I think it makes sense to make the parameter list match the docstring.